.github/workflows/native-build.yml

```diff
@@ -131,27 +131,20 @@ jobs:
     container:
       image:  ${{ matrix.run.container || '' }}
     steps:
-      - name: "Fix environment"
-        run: |
-          # https://github.com/actions/checkout/issues/1809
-          # https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/
-          echo "ACTIONS_RUNNER_FORCED_INTERNAL_NODE_VERSION=node16" >> $GITHUB_ENV
-          echo "ACTIONS_RUNNER_FORCE_ACTIONS_NODE_VERSION=node16" >> $GITHUB_ENV
```

Comment:
@LeStarch these links seemed specific to older versions of the `actions/checkout` et al.

Please confirm this is ok to remove, if it passes CI

Reply:
nvm it doesn't work, I had to revert
See note in https://github.com/nasa/fpp/pull/521/commits/1407109cc8125315dfff66f8925800fa2bae9fde